packages/integration-karma/test/accessibility/LightningElement.focus/index.spec.js

```diff
@@ -8,15 +8,27 @@ beforeEach(() => {
     document.body.focus();
 });
 
-// TODO [#1327]: enable after patching focus method
-xit('should focus the first internally focusable element (delegatesFocus=true)', () => {
-    const elm = createElement('x-focus', { is: DelegatesFocusTrue });
-    document.body.appendChild(elm);
-
-    elm.focus();
-    const input = elm.shadowRoot.querySelector('.delegates-focus-true-first');
-    expect(elm.shadowRoot.activeElement).toBe(input);
-});
+// TODO [#2126]: Firefox does not transfer focus to the host element when delegatesFocus is true.
```

Comment:
This comment is missing the fact that it's a FF gap in their shadow DOM implementation.

```suggestion
// TODO [#2126]: Firefox doesn't implements delegatesFocus natively.
```